Tensile/SolutionStructs.py

```diff
@@ -2951,6 +2953,8 @@ def assignDerivedParameters(state):
     bufferLoad = state["BufferLoad"] and state["KernelLanguage"] == "Assembly"
     if not bufferLoad:
       state["DirectToLds"] = False
+      state["DirectToLdsA"] = False
+      state["DirectToLdsB"] = False
```

Comment:
Should this combo be a reject to avoid duplicate kernels?

Reply:
I quickly tried DirectToLds=[0,1] + BufferLoad=[0,1].
Tensile did not generate duplicated kernels.
BL=0+DTL=1 case, DTL is changed to 0 and this kernel is not generated probably because all kernel parameters are exactly same as BL=0 + DTL=0.